### PR TITLE
Moved the variables to new file, and edited README.md

### PR DESCRIPTION
**Pull request by annettomcy**
Moved the variables from "src/sbuttons.less" to file " src/_variables.less " and imported it in the sbuttons.less file. Also edited the README.md accordingly.

Issue: Move variables in src/sbuttons.less to new file src/_variables.less #395

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,16 +40,16 @@ And then you can import the CSS file found in `dist/sbuttons.css` or the LESS fi
...
 ### Modifying Button Colors
 
-To modify button colors, import `src/sbuttons.less` in your Less file, then make changes to the variables after the import.
+To modify button colors, import `src/_variables.less` in your Less file, then make changes to the variables after the import.
 
 For example, to change the blue color to a different shade
 
 ```
-@import '/path/to/sbuttons.less';
+@import '/path/to/_variables.less';
```
Comment (shahednasser) [inline README.md:48]: Please don't change this.

File: README.md [hunk 2]
```diff
@@ -40,16 +40,16 @@ And then you can import the CSS file found in `dist/sbuttons.css` or the LESS fi
 
 ### Modifying Button Colors
 
-To modify button colors, import `src/sbuttons.less` in your Less file, then make changes to the variables after the import.
+To modify button colors, import `src/_variables.less` in your Less file, then make changes to the variables after the import.
```
Comment (shahednasser) [inline README.md:43]: Please don't change this

**Review by shahednasser — CHANGES_REQUESTED** [review]
Can you run `npm run compile` after you finish to see if anything in the website is affected by this change?

File: README.md (hunk 1 above)
Comment (annettomcy) [reply README.md:48]: Okay, will revert that

File: README.md (hunk 2 above)
Comment (annettomcy) [reply README.md:43]: Okay will revert that

**Comment by annettomcy** [comment]
Okay will run npm run compile after I finish.

**Comment by shahednasser** [comment]
@annettomcy did you run `npm run compile`? where there any changes?

**Comment by shahednasser** [comment]
Hi @annettomcy this is an urgent issue (as it has priority high) and other PRs are dependant on it. Can you finish this by today? If not I need to reassign it quickly.

**Comment by annettomcy** [comment]
Yes I ran npm run compile and the website worked fine, are there any more changes?

**Comment by shahednasser** [comment]
I cloned your repository and ran `npm run compile` and there were a lot of changes. Can you commit the changes you find after running it?

**Comment by annettomcy** [comment]
Hey are the changes visible now

**Comment by annettomcy** [comment]
Hey are the changes visible

File: package.json [hunk 3]
```diff
@@ -31,7 +31,7 @@
     "minimist": "^1.2.3",
     "node-less": "^1.0.0",
     "onchange": "^7.0.2",
-    "prettier": "^2.1.1",
+    "prettier": "^2.1.2",
```
Comment (shahednasser) [inline package.json:34]: Please remove this change

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by shahednasser** [comment]
Closing this as you took a long time on a high priority issue. Thank you.

**Pull request closed without merging**